crates/hir_def/src/nameres/tests/block.rs

```diff
@@ -70,3 +70,32 @@ fn outer() {
         "#]],
     );
 }
+
+#[test]
+fn nested_blocks() {
+    check_at(
+        r#"
+//- /lib.rs
```

Comment:
/lib is inferred for single-file fixtures

Reply:
oh, right. removed them in https://github.com/rust-analyzer/rust-analyzer/pull/7377